recipes/snippy/meta.yaml

```diff
@@ -12,36 +12,33 @@ source:
   sha256: '{{ sha256 }}'
 
 build:
-  number: 1
+  number: 0
   noarch: generic
 
 requirements:
   run:
-    - perl >=5.26
+    - perl
     - perl-bioperl >=1.7.2
-    - perl-time-piece
     - bwa >=0.7.17
-    # version comparison bug in 4.4.5, drop the maximum version pins
-    # when new version is released
-    - samtools >=1.9,<1.10
-    - bcftools >=1.9,<1.10
+    - samtools >=1.10
+    - bcftools >=1.10
     - bedtools >=2.28.0
     - parallel >=20170422
     - freebayes >=1.3.1
     - vcflib >=1.0.0_rc3
     - snpeff >=4.3
     - vt >=0.5772
     - snp-sites >=2.4
-    - emboss >=6.0
+    - any2fasta >=0.4
     - samclip >=0.2
-    - seqtk >=1.2
-    - minimap2 >=2.10
+    - seqtk >=1.3
+    - minimap2 >=2.17
     - openjdk =8
 
 test:
   commands:
-    - snippy --version
-    - snippy-core --version
+    - snippy --version 2>&1 | grep -F '{{ version }}'
```

Comment:
I don't know if that intentional or not, but `snippy --version` does not print to `stdout`: https://github.com/tseemann/snippy/blob/v4.5.0/bin/snippy#L672
(IMHO: Since the user requested that the version is printed, it would make more sense to print to `stdout`.)

Reply:
@mbargull how embarassing. i evangalise exactly what you suggest, but my own software isn;t doing it. not sure how i let that happen. thank you!

Reply:
@dpryan79 thank you for patching the test.